src/main/java/org/broadinstitute/hellbender/cmdline/PicardCommandLineProgram.java

```diff
@@ -35,20 +36,36 @@
     public Object instanceMain(final String[] argv) {
         // set general SAM/BAM parameters
         SamReaderFactory.setDefaultValidationStringency(VALIDATION_STRINGENCY);
+
+        // defer to parent
+        return super.instanceMain(argv);
+    }
+
+    /**
+     * Create a common SAMFileWriter for use with Picard tools.
+     *
+     * @param outputFile    - if this file has a .cram extension then a reference is required. Can not be null.
+     * @param referenceFile - the reference source to use. Can not be null if a output file has a .cram extension.
+     * @param header        - header to be used for the output writer
+     * @param preSorted     - if true then the records must already be sorted to match the header sort order
+     * @return SAMFileWriter
+     */
+    protected SAMFileWriter createCommonSAMWriter(
```

Comment:
Rename to `createSAMWriter()` for consistency with the post-code-review `GATKTool` version.